walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,120 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
 
+/*
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
     throw new UnsupportedOperationException("TODO: Implement this method.");
   }
-}
+}*/
+
+
+public final class FindMeetingQuery {
+  public boolean hasMatch(HashSet<String> first, HashSet<String> second) {
+    Iterator<String> it = first.iterator();
+    while(it.hasNext()){
+      if (second.contains(it.next())) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  public ArrayList<TimeRange> filterTimes(Collection<Event> events, HashSet<String> attendees) {
+    ArrayList<TimeRange> times = new ArrayList<TimeRange>();
+    for (Event event: events) {
+      HashSet<String> curr_attendees = new HashSet<String>();
```

Comment:
Use camelCase naming to keep it similar to rest of the variable names

Reply:
okay, changed `curr_attendees` to `currAttendees`